.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: generic
+
+sudo: required
+env:
+  - DOCKER_VERSION=1.12.1
+
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+    - $HOME/.m2
+
+before_install:
+  - curl -sSL https://get.docker.com/ | sh
+  - sudo apt-get -y --force-yes install docker-engine=$DOCKER_VERSION-0~precise
+  - sudo docker build --tag=build-image .
+
+script:
+  - sudo docker run -v $(pwd):/build -v $HOME/.m2:/root/.m2 -v $HOME/.gradle:/root/.gradle -v /var/run/docker.sock:/var/run/docker.sock build-image test --info
```

Comment:
Why do you want to execute the Gradle in Docker? Do you expect some integration tests working with real Postgres? Then it's OK...